contracts/SimpleEscrowEvents/simple_escrow_events.sol

```diff
@@ -0,0 +1,16 @@
+pragma solidity ^0.4.0;
+contract SimpleEscrowEvents {
+
+    event Payment(string indexed contractAddress, uint amount);
```

Comment:
here is our new Solidity contract. All this does is expose this `Event` as our interface.
I will be redeploying a new contract soon which also includes the destination stellar address and then we can probably consider that to be our final `API` for this `send()` function